browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1060,7 +1060,7 @@ L.CanvasTileLayer = L.Layer.extend({
 				setTimeout(this.update.bind(this), 200);
 			}, this._painter);
 		}
-		else {
+		else if (this._docType !== 'spreadsheet') { // See scroll section. panBy is used for spreadsheets while scrolling.
 			this._map.on('movestart', this._painter.startUpdates, this._painter);
```

Comment:
The profile suggested that both startUpdates and stopUpdates did a synchronous full re-draw during their execution ;-) - which I guess is not what is/was intended. Is that still so for spreadsheets ? =)

Reply:
It was working for spread sheets and updates were starting and stopping immediately without another function in between. So i cancelled it. I will also check the behaviour for Writer and others. It can be removed or improved i guess.

Also for scrolling, another functionality is used for Calc, so we don't seem to need this.